lib/utils/ensure-single-browser-tab-only.js

```diff
@@ -36,14 +36,14 @@ if (!foundElectron) {
     }
   });
 
-  window.addEventListener('pagehide', unload);
-  window.addEventListener('beforeunload', unload);
-
+  const eventName = /iPad|iPhone|iPod/.test(navigator.userAgent)
+    ? 'pagehide'
+    : 'beforeunload';
```

Comment:
would it be wise to perform feature-detection here instead of user-agent guessing?

```js
const eventName = 'undefined' !== typeof window.onbeforeunload
	? 'beforeunload'
	: 'pagehide'
```